site/app/views/submission/HomeworkView.php

```diff
@@ -476,6 +476,7 @@ private function renderSubmitBox(Gradeable $gradeable, $graded_gradeable, $versi
             'show_no_late_submission_warning' => !$gradeable->isLateSubmissionAllowed() && ($gradeable->isSubmissionClosed() && !$gradeable->isTaGradeReleased()),
             // This is only used as a placeholder, so the who loads this page is the 'user' unless the
             //  client overrides the user
+            'bulk_upload_access' => $this->core->getUser()->getGroup() <= $gradeable->getMinGradingGroup(),
```

Comment:
Should this check also be updated to `$this->core->getAccess()->canI("grading.electronic.grade", ["gradeable" => $gradeable]);`